.gitignore

```diff
@@ -0,0 +1,188 @@
+schema/
+src/main/java/th/or/etda/
+
+/
+/node_modules
+/RUNNING_PID
+/conf/localhost.conf
+
+### Vim example.template
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+*~
+### SBT example.template
```

Comment:
I think this .gitignore is way too long. maybe put in only what is needed

Reply:
fixed